moped-editor/src/utils/signalComponentHelpers.js

```diff
@@ -42,72 +38,25 @@ export const signalToFeatureCollection = signal => {
     / geometry will be saved.
     */
     const featureUUID = signal?.id || uuidv4();
-    const feature = {
-      type: "Feature",
-      properties: {
-        ...signal?.properties,
-        renderType: "Point",
-        // I was considering changing this to be INTERSECTIONID, but since the sourceLayer is being saved
-        // as "drawnByUser" and not "ATD_ADMIN.CTN_Intersections", I've left it as PROJECT_EXTENT_ID
-        PROJECT_EXTENT_ID: featureUUID,
-        sourceLayer: "drawnByUser",
+    const featureSignalsRecord = {
+      // MultiPoint coordinates are an array of arrays, so we wrap the coordinates
+      geography: {
+        ...signal.geometry,
+        type: "MultiPoint",
+        coordinates: [signal.geometry.coordinates],
       },
-      geometry: signal.geometry,
-      id: featureUUID,
+      knack_id: signal.properties.id,
+      location_name: signal.properties.location_name,
+      signal_type: signal.properties.signal_type,
+      signal_id: signal.properties.signal_id,
+      // I was considering changing this to be INTERSECTIONID, but since the sourceLayer is being saved
+      // as "drawnByUser" and not "ATD_ADMIN.CTN_Intersections", I've left it as PROJECT_EXTENT_ID
+      project_extent_id: featureUUID,
+      source_layer: "drawnByUser",
```

Comment:
I left these comments so that we can rethink the source layer name for the signal features. 

It doesn't seem as important as it once was since we can now get the name of the table that a feature is stored in from the component record itself.

Reply:
i think we should drop these two columns unless you have a need for them in your code @mddilley. or i can earmark it as something to do after we run the transpose geometry script for the last time.

Reply:
😎 Thanks, sounds great to me - I'll remove them in this PR and go through the test steps to make sure it doesn't break anything. I think that it will be good to take care of it now.